log/log_test.go

```diff
@@ -103,125 +108,47 @@ func (s *RelayTestSuite) TearDownSuite(t *C) {
 // Test cases
 // //////////////////////////////////////////////////////////////////////////
 
+// @TODO: Log level doesn't work; It was broken with the first commit to this repo
+// Below was in relay.go, so logs below log level were skipped
+// case entry := <-r.logChan:
+// 	Skip if log level too high, too verbose.
+// 	if entry.Level > r.logLevel {
+// 		continue
+// 	}
+// above is missing now, so all logs are registered
```

Comment:
This functionality seems to be completely broken. Probably not used. Read the comment.